db/db.go

```diff
@@ -314,19 +320,13 @@ func createTables(MetadataDbClient MetadataStorage) error {
 		connection_id VARCHAR NOT NULL,
 		consumers_group VARCHAR NOT NULL,
 		max_ack_time_ms SERIAL NOT NULL,
-		created_by INTEGER,
-		created_by_username VARCHAR NOT NULL,
 		is_active BOOL NOT NULL DEFAULT true,
-		created_at TIMESTAMPTZ NOT NULL,
-		is_deleted BOOL NOT NULL DEFAULT false,
+		updated_at TIMESTAMPTZ NOT NULL,
```

Comment:
move to end of table, when upgraded this column will be created at the end from the alter table and can cause inconsistencies
also change the struct